package.json

```diff
@@ -29,9 +30,12 @@
     "node": ">= 0.10"
   },
   "dependencies": {
+    "@types/bluebird": "~3.0.36",
+    "@types/express": "~4.0.34",
```

Comment:
Just to be clear: I think they should probably be moved to `devDependencies` (if this works in production as well)